code/datums/mind.dm

```diff
@@ -67,118 +67,140 @@
-	var/mob/living/old_current = current
-	if(current)
-		current.transfer_observers_to(new_character)	//transfer anyone observing the old character to the new one
-	current = new_character								//associate ourself with our new body
-	new_character.mind = src							//and associate our new body with ourself
-	for(var/a in antag_datums)	//Makes sure all antag datums effects are applied in the new body
-		var/datum/antagonist/A = a
-		A.on_body_transfer(old_current, current)
-	if(iscarbon(new_character))
-		var/mob/living/carbon/C = new_character
-		C.last_mind = src
-	transfer_antag_huds(hud_to_transfer)				//inherit the antag HUD
-	transfer_actions(new_character)
-	transfer_martial_arts(new_character)
-	RegisterSignal(new_character, COMSIG_MOB_DEATH, .proc/set_death_time)
-	if(active || force_key_move)
-		new_character.key = key		//now transfer the key to link the client to our new body
-	current.update_atom_languages()
-
-
-///Adjust experience of a specific skill
-/datum/mind/proc/adjust_experience(skill, amt, silent = FALSE)
-	var/datum/skill/S = GetSkillRef(skill)
-	skill_experience[S] = max(0, skill_experience[S] + amt) //Prevent going below 0
-	var/old_level = known_skills[S]
-	switch(skill_experience[S])
-		if(SKILL_EXP_LEGENDARY to INFINITY)
-			known_skills[S] = SKILL_LEVEL_LEGENDARY
-		if(SKILL_EXP_MASTER to SKILL_EXP_LEGENDARY)
-			known_skills[S] = SKILL_LEVEL_MASTER
-		if(SKILL_EXP_EXPERT to SKILL_EXP_MASTER)
-			known_skills[S] = SKILL_LEVEL_EXPERT
-		if(SKILL_EXP_JOURNEYMAN to SKILL_EXP_EXPERT)
-			known_skills[S] = SKILL_LEVEL_JOURNEYMAN
-		if(SKILL_EXP_APPRENTICE to SKILL_EXP_JOURNEYMAN)
-			known_skills[S] = SKILL_LEVEL_APPRENTICE
-		if(SKILL_EXP_NOVICE to SKILL_EXP_APPRENTICE)
-			known_skills[S] = SKILL_LEVEL_NOVICE
-		if(0 to SKILL_EXP_NOVICE)
-			known_skills[S] = SKILL_LEVEL_NONE
-	if(isnull(old_level) || known_skills[S] == old_level)
-		return //same level or we just started earning xp towards the first level.
-	if(silent)
-		return
-	if(known_skills[S] >= old_level)
-		S.level_gained(src, known_skills[S], old_level)
-	else
-		S.level_lost(src, known_skills[S], old_level)
-
-///Gets the skill's singleton and returns the result of its get_skill_modifier
-/datum/mind/proc/get_skill_modifier(skill, modifier)
-	var/datum/skill/S = GetSkillRef(skill)
-	return S.get_skill_modifier(modifier, known_skills[S] || SKILL_LEVEL_NONE)
-
-/datum/mind/proc/get_skill_level(skill)
-	var/datum/skill/S = GetSkillRef(skill)
-	return known_skills[S] || SKILL_LEVEL_NONE
-
-/datum/mind/proc/print_levels(user)
-	var/list/shown_skills = list()
-	for(var/i in known_skills)
-		if(known_skills[i]) //Do we actually have a level in this?
+	init_known_skills() 
+ 
+/datum/mind/Destroy() 
+	SSticker.minds -= src 
+	if(islist(antag_datums)) 
+		QDEL_LIST(antag_datums) 
+	return ..() 
+ 
+/datum/mind/proc/get_language_holder() 
+	if(!language_holder) 
+		language_holder = new (src) 
+	return language_holder 

```

Comment:
You seem to be adding a space to every line for some reason, increasing the diff.
Can you fix that?
```suggestion
/datum/mind/Destroy()
	SSticker.minds -= src
	if(islist(antag_datums))
		QDEL_LIST(antag_datums)
	return ..()

/datum/mind/proc/get_language_holder()
	if(!language_holder)
		language_holder = new (src)
	return language_holder
```
Same all around the file.

Reply:
This is still a thing.

Reply:
Doom, why...would you put spaces? I mean, it's a small detail but these look more intentional than just a typo...I'm confus

Reply:
This is not an intentional change. It happens with EoL changes.

Reply:
Fixed it, sorry - didn't notice your comments until just now